about.html

```diff
@@ -0,0 +1,239 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="./css/about.css" />
+
+    <title>World Tourism Summit 2023</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <header>
+      <!--    TOP BAR    -->
+      <div class="header__top-bar">
+        <div class="header__top-bar__container">
+          <a href="#"
+            ><img src="./assets/ico/facebook.png" alt="Facebook logo"
+          /></a>
+          <a href="#"
+            ><img src="./assets/ico/twitter.png" alt="Twitter logo"
+          /></a>
+          <a href="#">English</a>
+          <a href="#">My Page</a>
+          <a href="#">Logout</a>
+        </div>
+      </div>
+
+      <!--    NAVBAR CONTAINER   -->
+      <nav>
+        <div id="navbar" class="navbar">
+          <!-- Summit logo-->
+          <div class="summit__logo">
+            <a href="#">
+              <img src="./assets/ico/world.png" alt="World icon" />
+              <div class="summit__logo_text">
+                <h4 class="summit__logo__up">World Tourism</h4>
+                <h3 class="summit__logo__mid">SUMMIT</h3>
+                <h4 class="summit__logo__bot">2023</h4>
+              </div>
+            </a>
+          </div>
+
+          <!-- Navbar links -->
+          <ul class="navbar__links">
+            <li><a href="index.html" onclick="toggleMenu()">Home</a></li>
+            <li><a href="about.html" onclick="toggleMenu()">About</a></li>
+            <li>
+              <a href="index.html#program" onclick="toggleMenu()">Program</a>
+            </li>
+            <li>
+              <a href="index.html#speakers" onclick="toggleMenu()">Speakers</a>
+            </li>
+            <li>
+              <a href="index.html#sponsors__section" onclick="toggleMenu()"
+                >Partners</a
+              >
+            </li>
+            <li>
+              <a href="index.html" onclick="toggleMenu()"
+                >World Tourism Summit</a
+              >
+            </li>
+          </ul>
+
+          <!-- hamburger menu -->
+          <button id="burger" class="navbar__menu" type="button">
+            <span class="bar top"></span>
+            <span class="bar mid"></span>
+            <span class="bar bot"></span>
+          </button>
+        </div>
+      </nav>
+    </header>
+
+    <!--    START HERE -->
+    <main>
+      <div class="about__main__hero">
+        <section class="about__main__section">
+          <h2 class="about__main__section__subtitle">
+            "Hello! Be part of the change"
+          </h2>
```

Comment:
- [ ] To correctly match the template design, kindly ensure that you remove this text on your about page as it is not part of the original template. Below you can see the headline of the About to the original template
![Screenshot from 2023-05-25 03-14-00](https://github.com/sergio-usma/First_capstone_project/assets/83042742/f114ed57-8317-4d63-96a9-f2c99f3ef447)

Reply:
Ok! The text has been removed.